v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -103,9 +99,13 @@ def validate_globals_match(
     ht1_globals: Globals,
     ht2_globals: Globals,
     dataset: str,
+    validate_selects: bool = True,
 ) -> bool:
     results = []
     for field in dataclasses.fields(Globals):
+        if field.name == 'selects' and not validate_selects:
```

Comment:
I added this because crdq queries are more like annotations table queries and will never match the rdc or raw dataset selects